src/components/FinalCard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const FinalCardContainer = styled.div`
+  background-color: #fafafa;
```

Comment:
Maybe it's not that important but I would suggest to stick with one way to specify colors - in some places we use the names, as `black` and in others the code as `#fafafa` which doesn't look good. 
I prefer the codes because then we have more options.
If you agree, we can open an issue to change the names to codes and do it later, and from now on use only codes :art:

Reply:
Yeah lets just use codes I have been selecting them from [google material design colors](https://material.io/guidelines/style/color.html#color-color-palette)

Reply:
Perfect :+1: